.github/release-drafter.yml

```diff
@@ -0,0 +1,35 @@
+name-template: "$NEXT_MINOR_VERSION"
+tag-template: "$NEXT_MINOR_VERSION"
+change-template: '- $TITLE #$NUMBER [@$AUTHOR]'
+
+categories:
+  - title: "✨ Features"
+    labels: 
+      - "feature"
+      - "✨"
+      - "⚡️"
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
+      - 'bug'
+      - '🐛'
+  - title: "📚 Documentation"
+    labels:
+      - "documentation"
+      - "docs"
+      - "📚"
+  - title: " 🚨 Testing"
+    labels: 
+      - "testing"
+      - "test"
+
+exclude-labels:
+  - "changelog: skip"
+
+template: |
+
```

Comment:
intentional blank line?

Reply:
it was in the template I found, i haven't gone down to  the source code
```suggestion
```